### PR TITLE
fix(helm): bad indentation in bridge client example for gateway

### PR DESCRIPTION
**Pull request by passionne**
## Issue

https://gravitee.atlassian.net/browse/TT-5569

## Description

The documentation example added one more indentation than expected.

### CODE REVIEW

**Review by Okhelifi — APPROVED** [review]

File: helm/values.yaml [hunk 1]
```diff
@@ -1114,61 +1114,61 @@ gateway:
...
+      #  # in this case retry occurs like this: 2s, 3s, 4.5s, 6.7s, 10.1s, 15,2s etc.
+      #  connectionRetry:
+      #    delaySec: 2
+      #    maxDelaySec: 60
+      #    backoffFactor: 1.5 # how exponential we get to 60s (1.0 means linear)
+      #  authentication:
+      #    type: none # or basic or jwt
+      #    basic:
+      #      username: admin
+      #      password: adminadmin
```
Comment (graviteeio) [inline helm/values.yaml:1139]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       #      password: **********  
[This comment was created by Aqua Pipeline]

File: helm/values.yaml [hunk 2]
```diff
@@ -1114,61 +1114,61 @@ gateway:
...
+      #    jwt:
+      #      token: eyJhbGciOiJIUzI1NiI...
```
Comment (graviteeio) [inline helm/values.yaml:1141]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       #      token: **********************  
[This comment was created by Aqua Pipeline]

File: helm/values.yaml [hunk 3]
```diff
@@ -1114,61 +1114,61 @@ gateway:
...
+      #  ssl:
+      #    # defaults
+      #    ### beware: since 4.4 default is false (see upgrade guide)
+      #    trustAll: false
+      #    verifyHostname: true
+      #    # custom config for mTLS
+      #    keystore:
+      #      type: pkcs12 # can be jks / pkcs12 / pem
+      #      path: ${gravitee.home}/security/bridge/mtls-keystore.p12
+      #      password: s3cr3t
```
Comment (graviteeio) [inline helm/values.yaml:1151]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       #      password: ******  
[This comment was created by Aqua Pipeline]

File: helm/values.yaml [hunk 4]
```diff
@@ -1114,61 +1114,61 @@ gateway:
...
+      #      # for pem
+      #      # certPath: ${gravitee.home}/security/mtls-bridge-cert.pem
+      #      # keyPath: ${gravitee.home}/security/mtls-bridge-key.pem
+      #      # certContent: secret://...  # or raw pem, same for key content
```
Comment (graviteeio) [inline helm/values.yaml:1155]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       #      # certContent: secret:*****************************************  
[This comment was created by Aqua Pipeline]

File: helm/values.yaml [hunk 5]
```diff
@@ -1114,61 +1114,61 @@ gateway:
...
+      #      # keyContent:
+      #    # config for non public CAs
+      #    truststore:
+      #      type: pem # can be jks / pkcs12 / pem
+      #      path: ${gravitee.home}/security/bridge/rootCA.pem
+      #      # certContent: secret://...  # or raw pem, same for key content
```
Comment (graviteeio) [inline helm/values.yaml:1161]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       #      # certContent: secret:*****************************************  
[This comment was created by Aqua Pipeline]

File: helm/values.yaml [hunk 6]
```diff
@@ -1114,61 +1114,61 @@ gateway:
...
+      #      # for jks/pkcs12
+      #      # password:
+      #  proxy:
+      #    enabled: true
+      #    host: proxy.example.com
+      #    port: 8080
+      #    username: proxy
+      #    password: pa$$w0rd
```
Comment (graviteeio) [inline helm/values.yaml:1169]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       #    password: ********  
[This comment was created by Aqua Pipeline]

File: helm/values.yaml (hunk 1 above)
Comment (passionne) [reply helm/values.yaml:1139]: not critical a password. It's just an example

File: helm/values.yaml (hunk 2 above)
Comment (passionne) [reply helm/values.yaml:1141]: not a critical token. It's just an example

File: helm/values.yaml (hunk 3 above)
Comment (passionne) [reply helm/values.yaml:1151]: not a critical password. It's just an example

File: helm/values.yaml (hunk 4 above)
Comment (passionne) [reply helm/values.yaml:1155]: not a critical secret. It's just an example

File: helm/values.yaml (hunk 5 above)
Comment (passionne) [reply helm/values.yaml:1161]: not a critical secret. It's just an example

File: helm/values.yaml (hunk 6 above)
Comment (passionne) [reply helm/values.yaml:1169]: not a critical password. It's just an example

**Pull request merged**